connectors/src/lib/models/intercom.ts

```diff
@@ -0,0 +1,202 @@
+import {
+  CreationOptional,
+  DataTypes,
+  ForeignKey,
+  InferAttributes,
+  InferCreationAttributes,
+  Model,
+} from "sequelize";
+
+import { Connector, sequelize_conn } from "@connectors/lib/models";
+
+export class IntercomConnectorState extends Model<
+  InferAttributes<IntercomConnectorState>,
+  InferCreationAttributes<IntercomConnectorState>
+> {
+  declare id: CreationOptional<number>;
+  declare createdAt: CreationOptional<Date>;
+  declare updatedAt: CreationOptional<Date>;
+
+  declare lastGarbageCollectionFinishTime?: Date;
+
+  declare connectorId: ForeignKey<Connector["id"]>;
+}
+
+IntercomConnectorState.init(
+  {
+    id: {
+      type: DataTypes.INTEGER,
+      autoIncrement: true,
+      primaryKey: true,
+    },
+    createdAt: {
+      type: DataTypes.DATE,
+      allowNull: false,
+      defaultValue: DataTypes.NOW,
+    },
+    updatedAt: {
+      type: DataTypes.DATE,
+      allowNull: false,
+      defaultValue: DataTypes.NOW,
+    },
+    lastGarbageCollectionFinishTime: {
```

Comment:
Do you think you'll need this already?

Reply:
+1 if cargo culted, you might want to wait until you have the need as each connector is a bit different when it comes to garbage collecting

Reply:
I thought I'd need this as there's no webhook for the help center articles but indeed was cargo culted. I removed the table entirely for now and will add it only when confirmed I need something like that 👍🏻